packages/office-ui-fabric-react/src/components/pickers/examples/TagPicker.Basic.Example.tsx

```diff
@@ -28,104 +40,63 @@ const _testTags: ITag[] = [
   'yellow',
 ].map(item => ({ key: item, name: item }));
 
-export interface ITagPickerDemoPageState {
-  isPickerDisabled?: boolean;
-}
-
-export class TagPickerBasicExample extends React.Component<{}, ITagPickerDemoPageState> {
+export const TagPickerBasicExample: React.FunctionComponent = () => {
   // All pickers extend from BasePicker specifying the item type.
-  private _picker = React.createRef<IBasePicker<ITag>>();
-
-  constructor(props: {}) {
-    super(props);
-    this.state = {
-      isPickerDisabled: false,
-    };
-  }
-
-  public render() {
-    return (
-      <div className={rootClass}>
-        <Checkbox
-          styles={checkboxStyles}
-          label="Disable Tag Picker"
-          checked={this.state.isPickerDisabled}
-          onChange={this._onDisabledButtonClick}
-        />
-        Filter items in suggestions: This picker will filter added items from the search suggestions.
-        <TagPicker
-          removeButtonAriaLabel="Remove"
-          onResolveSuggestions={this._onFilterChanged}
-          getTextFromItem={this._getTextFromItem}
-          pickerSuggestionsProps={{
-            suggestionsHeaderText: 'Suggested Tags',
-            noResultsFoundText: 'No Color Tags Found',
-          }}
-          itemLimit={2}
-          disabled={this.state.isPickerDisabled}
-          inputProps={{
-            onBlur: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onBlur called'),
-            onFocus: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onFocus called'),
-            'aria-label': 'Tag Picker',
-          }}
-        />
-        <br />
-        Filter items on selected: This picker will show already-added suggestions but will not add duplicate tags.
-        <TagPicker
-          removeButtonAriaLabel="Remove"
-          componentRef={this._picker}
-          onResolveSuggestions={this._onFilterChangedNoFilter}
-          onItemSelected={this._onItemSelected}
-          getTextFromItem={this._getTextFromItem}
-          pickerSuggestionsProps={{
-            suggestionsHeaderText: 'Suggested Tags',
-            noResultsFoundText: 'No Color Tags Found',
-          }}
-          itemLimit={2}
-          disabled={this.state.isPickerDisabled}
-          inputProps={{
-            onBlur: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onBlur called'),
-            onFocus: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onFocus called'),
-            'aria-label': 'Tag Picker',
-          }}
-        />
-      </div>
-    );
-  }
-
-  private _getTextFromItem(item: ITag): string {
-    return item.name;
-  }
+  const picker = React.useRef<IBasePicker<ITag>>(null);
+  const [tagPicker, { toggle: toggleTagPicker }] = useBoolean(false);
```

Comment:
The previous name here was better (booleans should typically have names starting with `is` or similar)